static/app/components/replays/replayController.tsx

```diff
@@ -183,7 +206,7 @@ const ReplayControls = ({
         onClick={() => toggleSkipInactive(!isSkippingInactive)}
       />
 
-      <ReplayPlaybackSpeed speedOptions={speedOptions} isCompact={compactLevel > 0} />
+      <ReplayOptionsMenu speedOptions={speedOptions} />
```

Comment:
Since the 'Fast-forward inactivity' feature is inside the dropdown now we should remove the button on line 200 of this file. It's got the label "Fast-forward idle moments"